app/src/main/java/com/globant/calculatoronboardingkotlin/MainActivity.kt

```diff
@@ -1,12 +1,112 @@
+import com.example.calculatoronboardingkotlin.R
+import com.globant.calculatoronboardingkotlin.mvp.model.CalculatorModel
+import com.globant.calculatoronboardingkotlin.mvp.presenter.CalculatorPresenter
+import com.globant.calculatoronboardingkotlin.mvp.view.CalculatorView
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.DIVIDE
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.DOT
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.MINUS
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.MULTIPLY
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_EIGHT
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_FIVE
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_FOUR
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_NINE
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_ONE
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_SEVEN
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_SIX
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_THREE
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_TWO
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.NUMBER_ZERO
+import com.globant.calculatoronboardingkotlin.utils.Constants.Companion.PLUS
+import kotlinx.android.synthetic.main.activity_main.button_main_add
+import kotlinx.android.synthetic.main.activity_main.button_main_clear
+import kotlinx.android.synthetic.main.activity_main.button_main_delete_current_number
+import kotlinx.android.synthetic.main.activity_main.button_main_divide
+import kotlinx.android.synthetic.main.activity_main.button_main_dot
+import kotlinx.android.synthetic.main.activity_main.button_main_equals
+import kotlinx.android.synthetic.main.activity_main.button_main_multiply
+import kotlinx.android.synthetic.main.activity_main.button_main_number_eight
+import kotlinx.android.synthetic.main.activity_main.button_main_number_five
+import kotlinx.android.synthetic.main.activity_main.button_main_number_four
+import kotlinx.android.synthetic.main.activity_main.button_main_number_nine
+import kotlinx.android.synthetic.main.activity_main.button_main_number_one
+import kotlinx.android.synthetic.main.activity_main.button_main_number_seven
+import kotlinx.android.synthetic.main.activity_main.button_main_number_six
+import kotlinx.android.synthetic.main.activity_main.button_main_number_three
+import kotlinx.android.synthetic.main.activity_main.button_main_number_two
+import kotlinx.android.synthetic.main.activity_main.button_main_number_zero
+import kotlinx.android.synthetic.main.activity_main.button_main_subtraction
 
 class MainActivity : AppCompatActivity() {
 
+    private var presenter: CalculatorPresenter? = null
```

Comment:
Pls try not use null with kotlin, you can use lateinit private var presenter: CalculatorPresenter

Reply:
done, thanks